modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -0,0 +1,164 @@
+import * as t from 'io-ts';
+
+export interface CreateInvoiceParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface CreateDepositAddressParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface GetInvoicesParams {
+  id?: string;
+  status?: 'open' | 'closed';
+}
+
+export interface LightningWithdrawalParams {
+  amount: number;
+  address?: string;
+  sequenceId?: string;
```

Comment:
We do require `sequenceId` for withdrawals (and addresses, of course). But we can have our code randomly generate a sequenceId if none is provided, and we can generate an address automatically as well, as long as the request to BitGo's API has both those fields on it. Just wanted to make sure that's the plan.

Reply:
yes, that is the plan. The behavior should be to allow a user to specify if they like, and automatically do the right thing if they don't specify.